packages/driver/cypress/support/defaults.js

```diff
@@ -60,3 +60,25 @@ Cypress.Commands.add('originLoadUtils', (origin) => {
     Cypress.require('./utils')
   })
 })
+
+// this is here to validate a command log generated by the reporter
+Cypress.Commands.add('getCommandLogInReporter', (commandName, opts = { isHidden: false }) => {
```

Comment:
I don't think our driver tests should be accessing the Reporter. We could instead have a cy-in-cy test that verifies `{ log: false }` with protocol enabled doesn't show up in the inner Reporter.

Reply:
The click, type and dom_hitbox driver specs interact with the command log UI. If you feel strongly about removing, I think that is fine, but I don't think there is enough value to have a cy-in-cy test for this change. Its too slow and expensive.

Reply:
Yeah, those specs are the reason we had to turn on the [--runner-ui](https://github.com/cypress-io/cypress/blob/develop/.circleci/workflows.yml#L610) for the driver tests. Ideally, the driver tests should be able to run without displaying the runner UI. I think there is value in verifying a `log: false` log doesn't show up in the runner but I think that should reside elsewhere.

Reply:
was there a reason those weren't updated with the introduction of `--runner-ui` if these tests are a concern?

Reply:
FYI - I removed these [`b7c571b`](https://github.com/cypress-io/cypress/pull/28531/commits/b7c571befb9d2e3877fa2983d570ce18d7cff12a)